.github/workflows/deploy_on_release.yml

```diff
@@ -15,10 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest", "macos-14", "windows-latest"]
-        python-version: ["3.9", "3.11"]
-        exclude:
-          - os: "macos-14"
-            python-version: "3.9"  # not available for macos-14
+        python-version: ["3.10", "3.11", "3.12"]
```

Comment:
We test oldest & newest supported versions. No need to spend extra compute on 3.11.